modules/product/config/optional/core.entity_form_display.commerce_line_item.product_variation.add_to_cart.yml

```diff
@@ -0,0 +1,26 @@
+uuid: 4d6eaaee-5d0d-41e5-b100-da5451fad534
+langcode: en
+status: true
+dependencies:
+  config:
+    - commerce_order.commerce_line_item_type.product_variation
+    - core.entity_form_mode.commerce_line_item.add_to_cart
+  module:
+    - commerce_cart
+id: commerce_line_item.product_variation.add_to_cart
+targetEntityType: commerce_line_item
+bundle: product_variation
+mode: add_to_cart
+content:
+  quantity:
```

Comment:
Shouldn't we hide the quantity by default? (It used to be disabled by default before)